nomad/mock/acl.go

```diff
@@ -228,16 +233,17 @@ func ACLAuthMethod() *structs.ACLAuthMethod {
 		MaxTokenTTL:   maxTokenTTL,
 		Default:       false,
 		Config: &structs.ACLAuthMethodConfig{
-			OIDCDiscoveryURL:    "http://example.com",
-			OIDCClientID:        "mock",
-			OIDCClientSecret:    "very secret secret",
-			OIDCScopes:          []string{"groups"},
-			BoundAudiences:      []string{"audience1", "audience2"},
-			AllowedRedirectURIs: []string{"foo", "bar"},
-			DiscoveryCaPem:      []string{"foo"},
-			SigningAlgs:         []string{"bar"},
-			ClaimMappings:       map[string]string{"foo": "bar"},
-			ListClaimMappings:   map[string]string{"foo": "bar"},
+			JWTValidationPubKeys: []string{},
```

Comment:
Would it be better to have separate mocks for OIDC and JWT based auth methods?

Reply:
I think that's actually a good shout. Separated in [92ef601](https://github.com/hashicorp/nomad/pull/15918/commits/92ef60197693583d59358eae059909de0fe6c00c)